src/components/shared/MarkerTooltipContents.js

```diff
@@ -40,9 +40,14 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
         );
       }
       case 'DOMEvent': {
+        let latency;
+        if (data.timeStamp) {
+          latency = `${formatTimeLength(data.startTime - data.timeStamp)}ms`;
+        }
         return (
           <div className="tooltipDetails">
             {_markerDetail('type', 'Type', data.eventType)}
+            {data.timeStamp && _markerDetail('latency', 'Latency', latency)}
```

Comment:
nit: use `latency` instead of `data.timeStamp` as this is what we manipulate here.

So maybe here is how you can do this:
```javascript
  let latency = 0;
  if (data.timeStamp) {
    latency = data.startTime - data.timeStamp;
  }
  // later ...
  return (
    ...
    {latency && _markerDetail('latency', 'Latency', `${formatTimeLength(data.startTime - data.timeStamp)}ms`)}
    ...
  );
```

What do you think ?